BetterRechner/SimpleCalculator.vb

```diff
@@ -48,62 +49,49 @@ Module SimpleCalculator
             Loop
             promptUser = True
 
-            Console.WriteLine("Please enter desired function. Press Enter to finish calcultion.")
-            Console.WriteLine("1. product(*)")
-            Console.WriteLine("2. sum(+)")
-            Console.WriteLine("3. difference(-)")
-            Console.WriteLine("4. quotient(/)")
-            Console.WriteLine("5. exponent(^)")
-            mathFunction = Console.ReadLine()
+            Do
+                'removed multiple console writes - TJR
+                Console.WriteLine("Please enter desired function. Press Enter to finish calculation." _
+                                    & vbNewLine & "1. product(*)" _
+                                    & vbNewLine & "2. sum(+)" _
+                                    & vbNewLine & "3. difference(-)" _
+                                    & vbNewLine & "4. quotient(/)" _
+                                    & vbNewLine & "5. exponent(^)")
 
-            If mathFunction = "Sum" Or mathFunction = "Addition" Or mathFunction = "Add" _
+                mathFunction = Console.ReadLine()
+
+                If mathFunction = "Sum" Or mathFunction = "Addition" Or mathFunction = "Add" _
                 Or mathFunction = "+" Or mathFunction = "2" Then
-                Try
+
```

Comment:
try/catch here doesn't do anything since the user input would already have been successfully converted and stored.